pkg/relayer/cmd/cmd_start.go

```diff
@@ -18,7 +18,6 @@ import (
 	"github.com/pokt-network/poktroll/pkg/polylog"
```

Comment:
Leaving a general note.

Most RelayMiners have `:memory:` as the default in PROD right now.

We HAVE to make this be backwards compatible and log (announce) that it should be changed.